packages/mdc-fab/README.md

```diff
@@ -128,21 +128,40 @@ You can also do this declaratively when using the [material-components-web](../m
 
 FABs are fully aware of ripple styles, so no DOM or CSS changes are required to use them.
 
-## Classes
-
 ### Block
 
 The block class is `mdc-fab`. This defines the top-level button element.
 
 ### Element
+
 The button component has a single `span` element added as a child of the button due to buttons not adhering to flexbox rules
 in all browsers. See [this Stackoverflow post](http://stackoverflow.com/posts/35466231/revisions) for details.
 
-### Modifier
+### CSS Classes
+
+CSS Class | Description
+--- | ---
+`mdc-fab--mini` | Makes the fab smaller (40 x 40 pixels)
+`mdc-fab--plain` | Makes the FAB have a white background
+
+### Sass Mixins
+
+Mixin | Description
+--- | ---
+`mdc-fab-theme($config)` | Generates theme-related CSS properties from the given config map
+
+#### `mdc-fab-theme($config)`
+
+Generates theme-related CSS properties from the given config map.
```

Comment:
If we're going to repeat its name/description here, maybe it doesn't make sense to bother with the table above?

Reply:
Done!